projects/plugins/jetpack-boost/CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,28 @@
+# Contributor Code of Conduct
```

Comment:
You can probably drop this in favor of the one in the monorepo root.

Reply:
Deleted https://github.com/Automattic/jetpack/pull/20512/commits/20530e155c219bc6baf3899c5469f03dabbe4025